pages/platform/ai/training_tuto_07_train_rasa_chatbot/guide.en-gb.md

```diff
@@ -0,0 +1,210 @@
+---
+title: AI Training - Tutorial - Train a rasa chatbot with docker and AI Training 
```

Comment:
_I would have shortened the title (SEO):_

title: AI Training - Tutorial - Train a rasa chatbot inside Docker container